Headers/DebugServer2/Target/FreeBSD/Thread.h

```diff
@@ -28,8 +28,8 @@ class Thread : public ds2::Target::POSIX::Thread {
   Thread(Process *process, ThreadId tid);
 
 protected:
-  virtual ErrorCode updateStopInfo(int waitStatus);
-  virtual void updateState();
+  ErrorCode updateStopInfo(int waitStatus) override;
+  void updateState() override;
```

Comment:
Yeah I used clang-modernize and -Wsuggest-overrides and friends but that
only works for code we build with these tools. We rarely build FreeBSD :)

On Thursday, April 28, 2016, strager <notifications@github.com> wrote:

> In Headers/DebugServer2/Target/FreeBSD/Thread.h
> <https://github.com/facebook/ds2/pull/304#discussion_r61446388>:
>
> > @@ -28,8 +28,8 @@ class Thread : public ds2::Target::POSIX::Thread {
> >    Thread(Process *process, ThreadId tid);
> >
> >  protected:
> > -  virtual ErrorCode updateStopInfo(int waitStatus);
> > -  virtual void updateState();
> > +  ErrorCode updateStopInfo(int waitStatus) override;
> > +  void updateState() override;
>
> clang-modernize and/or a clang warning can help with this. =]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/ds2/pull/304/files/9d4c89d42f3eea5e955fd481091eec0f9266ac26..4a0b3af912e94f832b01451d432170ee02b0053a#r61446388>
>


-- 
Stephane Sezer